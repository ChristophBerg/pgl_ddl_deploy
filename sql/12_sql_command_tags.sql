SELECT pgl_ddl_deploy.sql_command_tags(NULL);

SELECT pgl_ddl_deploy.sql_command_tags('');

SELECT pgl_ddl_deploy.sql_command_tags('CREATE EXTENSON foo;');

SELECT pgl_ddl_deploy.sql_command_tags('CREATE TABLE foo(); ALTER TABLE foo ADD COLUMN bar text; DROP TABLE foo;');
