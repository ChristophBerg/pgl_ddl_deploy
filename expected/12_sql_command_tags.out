SELECT pgl_ddl_deploy.sql_command_tags(NULL);
 sql_command_tags 
------------------
 
(1 row)

SELECT pgl_ddl_deploy.sql_command_tags('');
ERROR:  Invalid sql command
SELECT pgl_ddl_deploy.sql_command_tags('CREATE EXTENSON foo;');
ERROR:  syntax error at or near "EXTENSON"
LINE 1: SELECT pgl_ddl_deploy.sql_command_tags('CREATE EXTENSON foo;...
               ^
SELECT pgl_ddl_deploy.sql_command_tags('CREATE TABLE foo(); ALTER TABLE foo ADD COLUMN bar text; DROP TABLE foo;');
              sql_command_tags               
---------------------------------------------
 {"CREATE TABLE","ALTER TABLE","DROP TABLE"}
(1 row)

